arch/x86/core/Kconfig.ia32

```diff
@@ -185,4 +185,12 @@ config X86_MFENCE_INSTRUCTION_SUPPORTED
 	  Set n to disable the use of MFENCE instruction in arch_dcache_flush()
 	  for X86 CPUs have CLFLUSH instruction but no MFENCE
 
+config RUNTIME_IRQ_STATS
```

Comment:
Please prefix this with `X86_`.

Reply:
good suggestion, just updated it, thanks.